tests/e2e/default/telemetry/telemetry.wdio-spec.js

```diff
@@ -190,11 +189,6 @@ describe('Telemetry', () => {
       expect(await getTelemetryEntryByKey('search_match:contacts_by_type_freetext:name')).to.have.lengthOf(2);
       expect(await getTelemetryEntryByKey('search_match:contacts_by_type_freetext:phone')).to.have.lengthOf(2);
 
-      const searchField = await $('.select2-search__field');
```

Comment:
Not sure exactly what it was that changed with this test.  I think it has always been a bit flaky around the timing of opening and shutting the search drop-down.  My guess is that something changed with the timing to exacerbate the flakiness (or I just got really unlucky).  Either way, I think my fix here is more stable anyway where I have updated the `generic-form` code to always "close" the search box when we clear the data from it (by clicking on the form title). This is the same way we close the date-picker.